io.appflowy.AppFlowy.yml

```diff
@@ -0,0 +1,34 @@
+app-id: io.appflowy.AppFlowy
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: app_flowy
+separate-locales: false
```

Comment:
any reason to do that?

Reply:
To be honest, I copied this code from another project.yml.